packages/amplication-client/src/Workspaces/queries/resourcesQuery.ts

```diff
@@ -0,0 +1,41 @@
+import { gql } from "@apollo/client";
+
```

Comment:
is this file for a single query or queries? should we call this resourceQueries?